tests/mmptest/src/FrameworkLinksTests.cs

```diff
@@ -114,5 +114,21 @@ void AssertLinkedFrameworkStatus (string[] clangParts)
 
 			AssertFrameworkMinOSRespected (status);
 		}
+
+
+		[Test]
+		public void ProjectWithLinkToPrivateFramework_ShouldBuild ()
+		{
+			RunMMPTest (tmpDir => {
+
+				TI.UnifiedTestConfig test = new TI.UnifiedTestConfig (tmpDir) { TestDecl = @"
+	const string MobileDeviceLibrary = ""/System/Library/PrivateFrameworks/MobileDevice.framework/MobileDevice"";
+	[System.Runtime.InteropServices.DllImport (MobileDeviceLibrary)]
+	private static extern void TestMethod ();
+				" };
```

Comment:
Beside the fact that it should build - should it be linking with a `-framework /System/Library/PrivateFrameworks/MobileDevice.framework/MobileDevice` ? 

That might be eventually bite us at some point.